src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './component/Calculator';
+import './component/App.css';
 
 function App() {
   return (
```

Comment:
- [ ] Weldone creating the `App` module and the `Calculator` module. However, It is required that the `App.js` component be class-based. Please let's make it class-based to meet the requirement 💯. You can check out how to do this [here](https://www.geeksforgeeks.org/reactjs-class-based-components/)

- [ ] I will add this requirement here because I think you didn't see it from the summary from the previous reviewer. So please remove all the boilerplate that got created by the CRA (texts, images, and styles). You may remove all from the `public` directory except the `index.html`.

<img width="80" alt="Loom_VWzzfYEwOp" src="https://user-images.githubusercontent.com/81468675/153083269-003eeadb-12ad-4212-937f-8aa2abd17921.png">